intro-Linux/ch_uploading_files.tex

```diff
@@ -2,23 +2,25 @@ \chapter{Uploading/downloading/editing files}
 
 \subsection{Uploading/downloading files}
 
-Detailed information about uploading/downloading files with ``scp'' (using an OS
-X or Linux client) or WinSCP (Windows client), see
-\url{http://hpc.ugent.be/userwiki/index.php/User:VscCopy}.
+To transfer files from and to the HPC, see
+\href{https://hpcugent.github.io/vsc_user_docs/pdf/intro-HPC-linux-gent.pdf#section.3.2}{chapter 3.2 of the HPC manual}.
```

Comment:
Same here: use proper labels rather than section numbers + take into account OS+site in URL you're linking to.